scene/gui/label.cpp

```diff
@@ -35,9 +35,15 @@
 
 void Label::set_autowrap(bool p_autowrap) {
 
+	const bool changed = (autowrap != p_autowrap);
```

Comment:
Maybe we can do this instead?
```
if (autowrap == p_autowrap) {
	return;
}
```

Reply:
Updated. I was afraid that this will change the previous behavior.

But now, it seems only to cancel the regeneration of word cache, which I think won't make a difference if `autowrap` is not changed.